src/builder/context.rs

```diff
@@ -19,6 +19,9 @@ use super::timer;
 use file_util::create_dir;
 use process_util::PROXY_ENV_VARS;
 
+pub const PROTECTED_DIRS: &'static [&'static str] = &[
+    "dev", "proc", "run", "sys", "tmp", "work",
```

Comment:
Are there any technical reasons to protect them? I.e. probably this kind of "shrinked" container won't be used for running commands anyway, so these mount ponts aren't going to be useful